snap/README.md

```diff
@@ -145,28 +143,48 @@ For details on the mapping of configuration options to Config options, please re
 
 ```
 [Service]
-service.boot-timeout            // Service.BootTimeout
-service.health-check-interval   // Service.HealthCheckInterval
-service.host                    // Service.Host
-service.server-bind-addr        // Service.ServerBindAddr
-service.port                    // Service.Port
-service.protocol                // Service.Protocol
-service.max-result-count        // Service.MaxResultCount
-service.max-request-size        // Service.MaxRequestSize
-service.startup-msg             // Service.StartupMsg
-service.request-timeout         // Service.RequestTimeout
+service.boot-timeout                    // Service.BootTimeout
+service.health-check-interval           // Service.HealthCheckInterval
+service.host                            // Service.Host
+service.server-bind-addr                // Service.ServerBindAddr
+service.port                            // Service.Port
+service.protocol                        // Service.Protocol
```

Comment:
`service.boot-timeout` and `service.protocol` aren't supported and are already removed from some other snaps readmes.

Reply:
updated - please see latest commit.